template/$PROJECT_NAME$/apps/$PROJECT_NAME$_web/test/support/integration_case.ex

```diff
@@ -0,0 +1,23 @@
+<%= MixTemplates.ignore_file_unless(assigns[:integration] == "hound") %>
+defmodule <%= @project_name_camel_case %>Web.IntegrationCase do
+  @moduledoc """
+  This module defines the test case to be used by
+  tests that require a full, in-browser integration
+  test.
+
+  Relies on `<%= @project_name_camel_case %>Web.ConnCase`.
+  """
+
+  use ExUnit.CaseTemplate
+
+  using do
+    quote do
+      use <%= @project_name_camel_case %>Web.ConnCase
+      use Hound.Helpers
+
+      @moduletag :integration
+
+      hound_session()
```

Comment:
We may want to leave a comment describing the configuration necessary to run the session headlessly. 

```
additional_capabilities: %{
          chromeOptions: %{ "args" => [
            "--user-agent=#{Hound.Browser.user_agent(:chrome)}",
            "--headless",
            "--disable-gpu"
            ]
          }
        }
```

Reply:
According to the docs, it looks like this is what you need to do:

```elixir
# Use Chrome in headless mode with ChromeDriver (default port 9515 assumed) 
config :hound, driver: "chrome_driver", browser: "chrome_headless"
```

Reply:
Ah, they finally merged that update then. It wasn't possible to do that when I first implemented it.